README.md

```diff
@@ -2,7 +2,7 @@
 
 ## About
 
-This demo reads multiple tag values from an eWON Flexy IO Server and POSTs them to an OSIsoft PI Server.
+This application connects tags from a Flexy to an OSIsoft PI dataserver.
```

Comment:
Note that line 37 below also mentions osisoft demo. You may have corrected this in a future commit but wanted to note it now.